### PR TITLE
Detekt issues

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1549)

### CODE REVIEW

File: analyzer/src/main/kotlin/DotNetSupport.kt [hunk 1]
```diff
@@ -136,24 +136,13 @@ class DotNetSupport(
...
     private fun getPackageReferenceJsonContent(packageReference: PackageReference): Pair<String, JsonNode> {
-        if (packageReferencesAlreadyFound.containsKey(
-                Pair(packageReference.id.name, packageReference.id.version)
-            )
-        ) {
-            return packageReferencesAlreadyFound[Pair(
-                packageReference.id.name,
-                packageReference.id.version
-            )]!!
+        packageReferencesAlreadyFound[Pair(packageReference.id.name, packageReference.id.version)]?.let {
```
Comment (mnonnenmacher) [inline analyzer/src/main/kotlin/DotNetSupport.kt]: Commit message nit: "particular"

File: analyzer/src/main/kotlin/DotNetSupport.kt [hunk 2]
```diff
@@ -23,6 +23,7 @@ package com.here.ort.analyzer
 import com.fasterxml.jackson.databind.JsonNode
 import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
 
+import com.here.ort.model.EMPTY_JSON_NODE
```
Comment (mnonnenmacher) [inline analyzer/src/main/kotlin/DotNetSupport.kt]: Commit message nit: "Simplify", "particular"

File: analyzer/src/main/kotlin/DotNetSupport.kt (hunk 1 above)
Comment (sschuberth) [reply analyzer/src/main/kotlin/DotNetSupport.kt]: Done.

File: analyzer/src/main/kotlin/DotNetSupport.kt (hunk 2 above)
Comment (sschuberth) [reply analyzer/src/main/kotlin/DotNetSupport.kt]: Done.

**Pull request merged**